eureka/src/main/java/com/linecorp/armeria/server/eureka/EurekaUpdatingListener.java

```diff
@@ -266,7 +283,7 @@ private static String vipAddress(@Nullable String vipAddress, String hostName, P
         if (!portWrapper.isEnabled()) {
```

Comment:
this check is still helpful as it avoids adding secureVipAddress when there's no secure port